PatchesAndAreasApi/V1/Gateways/PatchesGateway.cs

```diff
@@ -29,6 +30,19 @@ public PatchesGateway(IDynamoDBContext dynamoDbContext, ILogger<PatchesGateway>
             _logger = logger;
         }
 
+        [LogCall]
+        public async Task<List<PatchEntity>> GetAllPatchesAsync()
+        {
+            _logger.LogDebug($"Calling IDynamoDBContext.ScanAsync for all PatchEntity records");
+
+            var scanConfig = new ScanOperationConfig();
```

Comment:
Could you create scanConfig in the constructor? It would be slightly cleaner.

Reply:
I would agree if scanConfig were used elsewhere in PatchesGateway. I'd say it's highly unlikely that other full table scans are going to be implemented in the future though, so for this reason I'd keep it where it is. Thoughts?